plugins/kotlin/idea/tests/testData/intentions/convertConcatenationToBuildString/basic.kt.after

```diff
@@ -0,0 +1,11 @@
+// WITH_RUNTIME
+fun test(foo: String, bar: Int, baz: Int) {
+    val s = buildString {
+        append("foo=")
+        append(foo)
+        append(",bar=")
+        append(bar)
+        append(",baz=")
+        append(baz + baz)
```

Comment:
I don't like this behavior. Interpolation shouldn't have been converted. I believe that this would be much more readable:
```kotlin
fun test(foo: String, bar: Int, baz: Int) {
    val s = buildString {
        append("foo=$foo")
        append(",bar=$bar")
        append(",baz=${baz + baz}")
    }
}
```